src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -89,6 +89,9 @@ $3dTiles_Provider.prototype.b3dmToMesh = function b3dmToMesh(data, layer) {
                 mesh.material = new BasicMaterial();
             }
         };
+        if (result.extensions.CESIUM_RTC) {
+            result.scene.RTC = result.extensions.CESIUM_RTC.center;
```

Comment:
Or maybe do this in the loader. This it'll work even when using the loader directly.